### PR TITLE
Update SEISMIC_SENSOR_ORIENTATION_METHODS.md

### PR DESCRIPTION
**Pull request by staylorofford**
Update orientation conventions and methods to reflect current understanding

### CODE REVIEW

**Comment by staylorofford** [comment]
Not critical, but bump regardless

**Comment by staylorofford** [comment]
bump

**Review by peanutty — APPROVED** [review]

File: docs/SEISMIC_SENSOR_ORIENTATION_POLARITIES.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+### Onsite
+
+When an orientation has been measured at the station, the `Azimuth` value will be this measurement and the `Method` will be `onsite`.
+
+The most common onsite orientation methods are using a compass and an azimuith pointing system. A compass measurement involves taking a bearing of the sensor's north component using a compass. An azimuth pointing system (APS) projects a laser onto the sensor's north component marking and uses differential GPS positioning of two antenna at either end of its laser to measure the sensor orientation. Both of these approaches are considered reliable and which method has been used to measure sensor orientation is not noted.
+
+### Offsite
+
+When an orientation is calculated using the data from a station or - as in the case of strong motion sensors in building arrays - using geographic references, the `Azimuth` value is the calculated value and the `Method` is `offsite`.
```
Comment (mnaguit) [inline docs/SEISMIC_SENSOR_ORIENTATION_POLARITIES.md:27]: Is the `offsite` for `Method` on of the available units for this metadata? If yes, we can include this in the README (https://github.com/GeoNet/delta/blob/main/install/README.md).

File: docs/SEISMIC_SENSOR_ORIENTATION_POLARITIES.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Polarities may describe the polarity of a single data stream (e.g., a Z component) or many data streams (e.g., all components). What the polarity value applies to is distinguished by the `Station,Location,Sublocation,Subsource` columns in the `polarities.csv`.
+
+### Measurement
+
+Polarity measurements are all data-based. Often polarities are communicated to GeoNet by data users noticing non-default values in their applications. The `Method,Citation` columns in `polarities.csv` captures the source of polarity information.
+
+## Combining and Communication Azimith and Polarity Information to Data Users
+
+Each seismic sensor component has an orientation and a polarity. GeoNet combines these as component orientations in StationXML as follows:
+1. The N or 1 component is described by the sensor azimuth, with the E/2 and Z component in a left-hand rule with respect to the N/1 component.
```
Comment (mnaguit) [inline docs/SEISMIC_SENSOR_ORIENTATION_POLARITIES.md:54]: Just a comment: structural arrays have XYZ components.

**Review by mnaguit — COMMENTED** [review]
Hi @staylorofford . Thanks for these updates.
Just a few comments from me (may or may not even need any change). 
I'll also tag @salichon in this PR.

File: docs/SEISMIC_SENSOR_ORIENTATION_POLARITIES.md (hunk 1 above)
Comment (staylorofford) [reply docs/SEISMIC_SENSOR_ORIENTATION_POLARITIES.md:27]: All orientations methods produce values in degrees, which go into Azimuth columns so... yes?

File: docs/SEISMIC_SENSOR_ORIENTATION_POLARITIES.md (hunk 2 above)
Comment (staylorofford) [reply docs/SEISMIC_SENSOR_ORIENTATION_POLARITIES.md:54]: Thanks, I will update

File: docs/SEISMIC_SENSOR_ORIENTATION_POLARITIES.md (hunk 1 above)
Comment (mnaguit) [reply docs/SEISMIC_SENSOR_ORIENTATION_POLARITIES.md:27]: Ah yeah, apologies. I was looking at the _Method_ field in _polarities.csv_ . I'm not so familiar with this new metadata yet, so I'm exploring the units. From the README, Method in _polarities.csv_ has these options: "study", "compass", or "unknown"

File: docs/SEISMIC_SENSOR_ORIENTATION_POLARITIES.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+The conventions and methods used by GeoNet to measure and represent seismic sensor orientations and polarities are outlined in this document.
+
+Data users encounter orientations and polarities through the component orientations in StationXML. This document describes the underlying conventions and processes which define these values.
+
+## Orientation Conventions and Measurement Methods
+
+Sensor orientations are contained in `delta.git/install/sensors.csv` under the `Azimuth` column. All sensor installations have an orientation entry, although it is not always defined.
+
+Seismic sensor orientations indicate the bearing of the sensor's north component and assume a left-hand convention for the orientation of the vertical and east components with respect to the north component (i.e., E component is at 90 degrees clockwise of N in the N-E plane; Z component is 90 degrees anticlockwise in N-Z plane).
```
Comment (ozym) [inline docs/SEISMIC_SENSOR_ORIENTATION_POLARITIES.md:11]: Not sure I understand what the Z component is in this context.

**Review by ozym — APPROVED** [review]
one small point but can be addressed later.

**Pull request merged**